news-buddy/src/components/ArticlesList.jsx

```diff
@@ -2,17 +2,19 @@ import { useState, useEffect } from "react"
 import ArticleCard from "./ArticleCard"
 import Loading from "./Loading"
 
-function ArticlesList({isLoading, setIsLoading}) {
+function ArticlesList() {
     const [articlesList, setArticlesList] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
 
     useEffect(() => {
+        setIsLoading(true)
         fetch('https://newsbuddy-f5h1.onrender.com/api/articles')
```

Comment:
We could extract out this logic into a separate api.js file which can house all of our api calls :)

Reply:
Hi Saima! Do you mean the fetch inside the useEffect could be extracted out?